tracer/test/Datadog.Trace.TestHelpers/EnvironmentVariablesCleanerAttribute.cs

```diff
@@ -0,0 +1,43 @@
+// <copyright file="EnvironmentVariablesCleanerAttribute.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Reflection;
+using Datadog.Trace.Util;
+using Xunit.Sdk;
+
+namespace Datadog.Trace.TestHelpers
+{
+    [AttributeUsage(AttributeTargets.Class, Inherited = true)]
+    public class EnvironmentVariablesCleanerAttribute : BeforeAfterTestAttribute
```

Comment:
I'm surprised this doesn't already exist - I could have sworn we were essentially doing this in some tests already!